pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -59,7 +59,8 @@ const unschedulableQTimeInterval = 60 * time.Second
 type SchedulingQueue interface {
 	Add(pod *v1.Pod) error
 	AddIfNotPresent(pod *v1.Pod) error
-	AddUnschedulableIfNotPresent(pod *v1.Pod) error
+	AddUnschedulableIfNotPresent(pod *v1.Pod, cycle int64) error
+	Cycle() int64
```

Comment:
Please rename this to SchedulingCycle. One could think that this is the queue cycle.